install_config/configuring_vsphere.adoc

```diff
@@ -235,3 +235,13 @@ Therefore, *_node-config.yaml_* must be in *_/etc/origin/node_* rather than
 [[vsphere-applying-configuration-changes]]
 == Applying Configuration Changes
 include::install_config/topics/applying_configuration_changes.adoc[]
+
+[[vsphere-backup]]
+== Backup of Persistent Volumes
+{product-title} provisions new volumes as *independent persistent disks* so
+it can freely attach and detach the volume any node in the cluster. As
```

Comment:
The first sentence is not clear. Should it be "{product-title} provisions new volumes as independent persistent disks to freely attach and detach the volume on any node in the cluster."

Reply:
I like your edit.

Reply:
Fixed